torch/quantization/_equalize.py

```diff
@@ -131,6 +142,11 @@ def converged(curr_modules, prev_modules, threshold=1e-4):
     if None in prev_modules.values():
         return False
     for name in curr_modules.keys():
-        difference = curr_modules[name].weight.sub(prev_modules[name].weight)
+        curr_weight = curr_modules[name].weight if type(
+            curr_modules[name]) in _supported_types else curr_modules[name][0].weight
```

Comment:
We seem to be using the logic `do a if x in _supported_types else b`. Maybe we can move this logic out to a helper function and invoke it in all the call sites where relevant?